### PR TITLE
fix font order

### PR DESCRIPTION
**Pull request by Louis-Matsika**
Resolves JIRA 1679

Overall changes
======
Added "Times New Roman" before Arial as our first backup font for reith qalam. Our current back upfont is too different from the main font, leading to bad content layout shift scores.

Code changes
======

- add `"Times New Roman"` on line 29 in `fontFamilies.ts`
- updated unit tests in `components/Text/index.test.tsx`
- fixed miss matching tests in `components/Paragraph/index.test.tsx`

Testing
======
**relevant unit tests:**
1. run `npx jest src/app/components/Paragraph/index.test.tsx -u`

**manual testing**
1. go to any service that uses reith qalam ([arabic for example](https://www.bbc.com/arabic/business-65050940))
2. go to any article 
3. inspect a paragraph
4. go to its inline style and under 'font-family' and remove `''BBC Reith Qalam"`
5. the paragraph should now fallback to times new roman

Helpful Links
======

[content layout shift](https://web.dev/cls/)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by eagerterrier — CHANGES_REQUESTED** [review]
tests need updating - see here for details https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635 

(relevant snippet below...)

```Summary of all failing tests
[29869](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29870)
 FAIL  src/app/components/Text/index.test.tsx
[29870](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29871)
  ● Text › should render the correct typography for the arabic service
[29871](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29872)

[29872](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29873)
    expect(element).toHaveStyle()
[29873](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29874)

[29874](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29875)
    - Expected
[29875](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29876)

[29876](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29877)
      color: rgb(20, 20, 20);
[29877](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29878)
    - fontFamily: "BBC Reith Qalam",Arial,Verdana,Geneva,Helvetica,sans-serif;
[29878](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29879)
    - fontSize: 0.9375rem;
[29879](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29880)
    - fontStyle: normal;
[29880](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29881)
    - fontWeight: 400;
[29881](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29882)
    - lineHeight: 1.5rem;
[29882](https://github.com/bbc/simorgh/actions/runs/4384894422/jobs/7676916635#step:10:29883)```

**Review by mfonofm — COMMENTED** [review]
Do we also need to make changes to `reithQalamFallback` in `psammead-styles > src > font-families`?

File: src/app/legacy/psammead/psammead-styles/src/font-families.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const helmetFontStyles = {
 /*
  *  BBC REITH QALAM
  */
-const reithQalamFallback = 'Arial, Verdana, Geneva, Helvetica, sans-serif;';
+const reithQalamFallback = '"Times New Roman", Arial, Verdana, Geneva, Helvetica, sans-serif;';
```
Comment (mfonofm) [inline src/app/legacy/psammead/psammead-styles/src/font-families.js]: nice, you'll just need to update the snapshots and run the linter to format your changes

**Review by eagerterrier — CHANGES_REQUESTED** [review]
Linting and then we’re done

**Review by shayneahchoon — APPROVED** [review]

**Review by eagerterrier — APPROVED** [review]
phew!

**Pull request merged**